content/swiftui/concepts/views/terms/button/button.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'Button'
+Description: 'A very basic component in the UI, which is used to interact and perform certain actions'
+Subjects:
+  - 'Sotfware Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Buttons'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **button** is a very basic component in the UI, which is used to interact and perform certain actions, such as logging into a page, or deleting a file etc.
+
+## Syntax
+
+```swift
+Button("Text inside the button"){
+  //What happens when you click the button
+}
+```
+
+The `Text inside the button` is what is shown inside the button.
+
+## Example
+
+To make a button which will `count` the number of times we `click` it:
+
+
+```swift
+@State var count = 0
+var body: some View {
+  Button("Click me: \(count)") {
+     self.count += 1
+  }
+ .font(.title)
+ .padding()
+ .backgroundColor(Color.green)
+ .cornerRadius(40)
+ .foregroundColor(Color.black)
+ .overlay(RoundedRectangle(cornerRadius: 40).stroke(Color.green))
+```
+The above snippet, creates a button with `Click Me` text. When you will click on it, the number next to the text will increase with the number of clicks.
+
+This will display the following:
+
+![Button](https://raw.githubusercontent.com/girijakar/docs/main/media/button.png)
```

Comment:
Can we link out to Codecademy/docs rather than your forked repo?

Reply:
Yes, I thought the same thing. But for that, I've to upload the file in the Codeacademy/docs.
How can I do that?

Reply:
@girijakar `https://raw.githubusercontent.com/Codecademy/docs/main/media/button.png`

Reply:
@Dusch4593 Thank you ☺